Resources/Prototypes/Recipes/Cooking/meal_recipes.yml

```diff
@@ -2384,3 +2384,30 @@
     FoodBreadCottonSlice: 2
     FoodCheeseSlice: 1
     FoodButterSlice: 1
+    
+- type: microwaveMealRecipe
+  id: RecipeBreakfastAmerican
+  name: full american breakfast recipe
+  result: FoodBreakfastAmerican
+  time: 30
+  group: Savory
+  reagents:
+    Egg: 12
+  solids:
+    FoodMeatBacon: 2
+    FoodPotato: 1
+    FoodBakedPancake: 2
+    
+- type: microwaveMealRecipe
+  id: RecipeBreakfastEnglish
+  name: full english breakfast recipe
+  result: FoodBreakfastEnglish
+  time: 30
+  group: Savory
+  reagents:
+    Egg: 12
+  solids:
+    FoodMeatBacon: 1
+    FoodMeat: 1
+    FoodTomato: 1
+    FoodSoybeans: 2 # replace these with green beans once those become a thing
```

Comment:
A Full English iiiiiis

* One or two slices of toast, optionally buttered
* Baked beans
* One or two eggs - fried or scrambled
* Sausage
* Bacon
* Black pudding
* Half of a beef tomato
* Mushrooms

What foodstuffs are we missing for this? Obviously black pudding (a type of spiced blood sausage)